config/locales/land_transaction_tax.en.yml

```diff
@@ -8,26 +8,26 @@ en:
     tooltip_hide: hide help
     tool_name: "land-transaction-tax-calculator"
     heading: "Land Transaction Tax (LTT) calculator"
-    heading_description: "Stamp Duty has been replaced by Land Transaction Tax (LTT) for properties bought in Wales. LTT will apply to residential properties costing over £180,000. You can use this calculator to find the amount of Land Transaction Tax you’ll have to pay.
-
-      You can also work out the cost of LTT on any additional properties you purchase over £40,000, like buy to lets or second homes, which will be charged a 3% additional supplement."
+    heading_description: "You must pay Land Transaction tax if you buy a property or land over a certain price in Wales over the current threshold of £180,000 for residential properties. Higher residential rates may apply if you already own one or more residential properties. However, if you are replacing your main residence the higher rates may not apply.  You must or arrange to send an LTT return to Welsh Revenue Authority and pay the tax within 30 days of the day after completion (or other effective date of the transaction)."
     heading_results: "Land Transaction Tax (LTT) calculator - Your Results"
     title: "Calculate the Land Transaction Tax on your residential property in Wales"
     next: Next
     select:
       label: I am
       option_prompt: please select an option
       option_isNextHome: buying my next home
-      option_isSecondHome: buying an additional or buy-to-let property
-      tooltip_html: "Anyone purchasing an additional home including buy to let properties over the value of £40,000 will have to pay an extra 3% surcharge on top of each Land Transaction Tax band."
+      option_isSecondHome: buying an additional property or second home
+      tooltip_html: "You will usually have to pay higher rates of Land Transaction Tax if you are buying a residential property worth %{amount} or more and already own 1 or more properties."
```

Comment:
We are not passing a value here for `%{amount}` so the view is rendering this literally. We should either ensure a value is passed form the view or we use the figure in the YAML.  
We also need to make this consistent across the English and Welsh language versions, which they are not as it stands (the Welsh language YAML contains the figure).

Reply:
This is still rendering as `%{amount}`, but in both languages now. It needs to be passed through from the view, like `text: t("land_transaction_tax.select.tooltip_html", amount: '£40,000')` in _calculator_form.html.erb